src/stories/InlineForm.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, StoryFn } from '@storybook/react';
+import styled from 'styled-components';
+import { IconQuestionCircleOutline } from '@taskany/icons';
+
+import { InlineForm } from '../components/InlineForm';
+import { Button } from '../components/Button';
+import { TableCell } from '../components/Table';
+import { Form } from '../components/Form';
+import { Popup } from '../components/Popup';
+import { FormInput } from '../components/FormInput';
+
+export default {
+    title: 'InlineForm',
+    component: InlineForm,
+    argTypes: {
+        onSubmit: { action: 'form submitted' },
+        onClick: { action: 'clecked' },
+        onMouseOver: { action: 'onMouseOver' },
+        onMouseLeave: { action: 'onMouseLeave' },
+        onChange: { action: 'onChange' },
+    },
+} as Meta<typeof InlineForm>;
+
+const StyledWrapper = styled.div`
+    display: contents;
```

Comment:
U already have this wrapper in InlineForm component. U are wrapping form twice in the same component.